core/Piranha/Models/ContentTypeField.cs

```diff
@@ -48,6 +48,11 @@ public sealed class ContentTypeField
         /// </summary>
         public string Description { get; set; }
 
+        /// <summary>
+        /// Gets/sets field order on the manager interface.
+        /// </summary>
+        public int SortOrder { get; set; }

```

Comment:
Why do we need to store this information? If sorting is applied when the content type is imported, it will then be serialized to `JSON` in the database and when retrieved fields will be in the correct order which should be the order in which the Manager UI will present them.

Reply:
in the example below the title field appears at the bottom on manager ui.

```
public class ProductProperties : CommonFields
{
      [Field]
      public ImageField Image { get; set; }

      [Field]
      public TextField Description { get; set; }
}

public class CommonFields
{
    [Field(Title = "Başlık")]
    public StringField Title { get; set; }
}
```